README.md

```diff
@@ -0,0 +1,26 @@
+# NewYorkTimes-Project
+
+# First project for Microverse HTML/CSS course
+
+# I'ts a copy of a New York Times article
+
+
+<img src="images/screenshot.png">
+
+Autor: 
+
+        -Jordy Garcia
+
+Languages:
+
+        -HTML
+        -CSS
+
+Live Preview:
+
+        https://garciajordy.github.io/NewYorkTimes-Project/
+
+Original Article:
+
+        https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0
+
```

Comment:
- This README is missing a lot of crucial sections. This was already pointed out in the previous review.
- Please reference [this](https://github.com/microverseinc/readme-template) template